apps/settings/test/marionette/app/regions/root.js

```diff
@@ -96,7 +99,7 @@ RootPanel.prototype = {
   // geolocation
   geolocation: function(enabled) {
     if (enabled !== this.geolocationCheckboxChecked) {
-      this.waitForElement('geolocationSpan').tap();
+      this.waitForElement('geolocationCheckbox').tap();
```

Comment:
.tap() seemed to be fine here, so maybe it's not needed? I think in most cases it probably doesn't matter...